draft-irtf-cfrg-hpke.md

```diff
@@ -950,11 +940,25 @@ def DeriveKeyPair(ikm):
   return (sk, pk(sk))
 ~~~
 
-where `order` is the order of the curve being used (this can be found in
-section D.1.2 of {{NISTCurves}}), and `bitmask` is defined to be 0xFF for P-256
-and P-384, and 0x01 for P-521. The precise likelihood of `DeriveKeyPair()`
-failing with DeriveKeyPairError depends on the group being used, but it
-is negligibly small in all cases.
+`order` is the order of the curve being used (see section D.1.2 of {{NISTCurves}}), and
+is listed below for completeness.
+
+~~~~~
+P-256:
+0xffffffff00000000ffffffffffffffffbce6faada7179e84f3b9cac2fc632551
+
+P-384:
+0xffffffffffffffffffffffffffffffffffffffffffffffffc7634d81f4372ddf
+  581a0db248b0a77aecec196accc52973
+
+P-521:
+0x01ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff
+  fa51868783bf2f966b7fcc0148f709a5d03bb5c9b8899c47aebb6fb71e91386409
+~~~~~
+
```

Comment:
In a new PR I use `order` one or two more times in different sections. This brings up the question of where, if anywhere, to include these constants. How important is it that these be here?

Reply:
I think it's fine to keep them here (for now). We can move them around if needed as a future editorial nit.